extensions/arc/src/localizedConstants.ts

```diff
@@ -59,6 +61,11 @@ export const yes = localize('arc.yes', "Yes");
 export const no = localize('arc.no', "No");
 export const feedback = localize('arc.feedback', "Feedback");
 export const selectConnectionString = localize('arc.selectConnectionString', "Select from available client connection strings below.");
+export const addingWokerNodes = localize('arc.addingWokerNodes', "adding worker nodes");
+export const workerNodesDescription = localize('arc.workerNodesDescription', "Expand your server group and scale your database by adding worker nodes.");
+export const coordinatorNodeDescription = localize('arc.coordinatorNodeDescription', "Custom coordinator node configuration is not available at this time. Your coordinator node configuration will mirror the worker node configuration selected above.");
+export const configurationInformation = localize('arc.configurationInformation', "You can configure number of vCores and storage size for worker node. Each worker node will have the same configuration. Adjust the number of vCores and memory settings for your server group.");
+export const workerNodesInformation = localize('arc.workerNodeInformation', "In preview it is not possible to reduce the number of worker nodes. Please refer to documentation for more perspectives.");
```

Comment:
Where is the link to this documentation? And I would suggest `for more information.` instead - that's the typical phrasing used for this kind of thing.

Reply:
The hyperlinks to documentation are found in the description section of this page.